doc/whats_new/v0.21.rst

```diff
@@ -107,6 +107,14 @@ Support for Python 3.4 and below has been officially dropped.
   when called before fit :issue:`12279` by :user:`Krishna Sangeeth
   <whiletruelearn>`.
 
+:mod:`sklearn.metrics`
+......................
+
+- |API| The metrics.calinski_harabaz_score has been renamed  to 
```

Comment:
redundant space between "renamed" and "to"
use :func: to link to corresponding function.